adal/src/telemetry/java/com/microsoft/aad/adal/EventStrings.java

```diff
@@ -0,0 +1,163 @@
+
+    static final String ACQUIRE_TOKEN_WITH_REFRESH_TOKEN_2 = "5";
+
+    static final String ACQUIRE_TOKEN_1 = "100";
+
+    static final String ACQUIRE_TOKEN_2 = "104";
+
+    static final String ACQUIRE_TOKEN_3 = "108";
+
+    static final String ACQUIRE_TOKEN_4 = "111";
+
+    static final String ACQUIRE_TOKEN_5 = "115";
+
+    static final String ACQUIRE_TOKEN_6 = "116";
+
+    static final String ACQUIRE_TOKEN_7 = "117";
+
+    // Event Parameter strings
+    static final String APPLICATION_NAME = "application_name";
+
+    static final String APPLICATION_VERSION = "application_version";
+
+    static final String CLIENT_ID = "client_id";
+
+    static final String CLIENT_IP = "client_ip";
+
+    static final String AUTHORITY_NAME = "authority";
+
+    static final String AUTHORITY_TYPE = "authority_type";
+
+    static final String AUTHORITY_TYPE_ADFS = "adfs";
+
+    static final String AUTHORITY_TYPE_AAD = "aad";
+
+    static final String API_DEPRECATED = "is_deprecated";
+
+    static final String AUTHORITY_VALIDATION = "authority_validation_status";
+
+    static final String PROMPT_BEHAVIOR = "prompt_behavior";
+
+    static final String EXTENDED_EXPIRES_ON_SETTING = "extended_expires_on_setting";
+
+    static final String WAS_SUCCESSFUL = "is_successful";
+
+    static final String AUTHORITY_VALIDATION_SUCCESS = "authority_validation_status_success";
+
+    static final String AUTHORITY_VALIDATION_FAILURE = "authority_validation_status_failure";
+
+    static final String AUTHORITY_VALIDATION_NOT_DONE = "authority_validation_status_not_done";
+
+    static final String CORRELATION_ID = "correlation_id";
+
+    static final String DEVICE_ID = "device_id";
+
+    static final String REQUEST_ID = "request_id";
+
+    static final String START_TIME = "start_time";
+
+    static final String STOP_TIME = "stop_time";
+
+    static final String RESPONSE_TIME = "response_time";
+
+    static final String REDIRECT_COUNT = "redirect_count";
+
+    static final String BROKER_APP = "broker_app";
+
+    static final String BROKER_VERSION = "x-broker_version";
+
+    static final String TOKEN_TYPE = "token_type";
+
+    static final String TOKEN_TYPE_IS_RT = "is_RT";
+
+    static final String TOKEN_TYPE_IS_MRRT = "is_MRRT";
+
+    static final String TOKEN_TYPE_IS_FRT = "is_FRT";
+
+    static final String TOKEN_TYPE_RT = "RT";
+
+    static final String TOKEN_TYPE_MRRT = "MRRT";
+
+    static final String TOKEN_TYPE_FRT = "FRT";
+
+    static final String CRYPTOGRAPHY_STATUS = "status";
+
+    static final String CRYPTOGRAPHY_EXCEPTION = "exception";
+
+    static final String HTTP_USER_AGENT = "user_agent";
+
+    static final String HTTP_METHOD = "method";
+
+    static final String HTTP_METHOD_POST = "post";
+
+    static final String HTTP_QUERY_PARAMETERS = "query_parameters";
+
+    static final String HTTP_RESPONSE_CODE = "response_code";
+
+    static final String HTTP_RESPONSE_METHOD = "response_method";
```

Comment:
Oops. forgot to delete

---
In reply to: [76171570](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/703#discussion_r76171570) [](ancestors = 76171570)